File: 0136-single-number/0136-single-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
   let obj={}
   for(let i=0;i<nums.length;i++){
        let value=nums[i]
        if(obj[value]){
            obj[value]++
        }
        else{
            obj[value]=1
        }
   }
   
   for(let key in obj){
    if(obj[key]==1){
        return Number(key)
    }
   }
   
};
console.log(singleNumber([2,2,1]));

File: 0136-single-number/0136-single-number.py
class Solution(object):
    def singleNumber(self, nums):
        count={}
        for i in range(len(nums)):
            value=nums[i]
            if value in count:
                count[value]+=1
            else:
                count[value]=1
        for key in count:
            if count[key]==1:
                return key